### DISCUSSION TITLE
How to un-install/remove Air?

### DISCUSSION THREAD

**Discussion by jesus-vc** [airsdk/Adobe-Runtime-Support#1350] [Q&A]
Hello, 

Would one of you mind helping me understand how to uninstall Air? It's located on a MacOS in the /Library/Frameworks/Adobe AIR.framework path. It appears as an SDK for Flash (4.7), and there is no installer or un-installer.

Thank you for any support you can provide!

**Comment by ajwfrost** [comment]
Do you know what version it is? If you view the contents of the Adobe AIR.framework and look in something like Versions/1.0/Resources there should be an Info.plist file, with an entry 'CFBundleVersion'.

Plus can you look under /Applications in case you see anything from Adobe there, or under /Applications/Utilities?

I believe with Adobe's version of AIR, there should be an uninstaller under /Applications/Utilities, and for our version of AIR there should be an /Applications entry which may be something like "Adobe AIR Installer" but can be run to uninstall it too.

thanks
thanks
